lib/config/presets/internal/default.ts

```diff
@@ -199,6 +199,23 @@ export const presets: Record<string, Preset> = {
         matchDepTypes: ['dependencies', 'require'],
         semanticCommitType: 'fix',
       },
+      {
+        matchDatasources: ['maven'],
+        matchDepTypes: ['test', 'build'],
+        semanticCommitType: 'chore',
```

Comment:
Isn't `chore` the default, so we can remove this rule?

Reply:
Yes, it is purely to be explicit

Reply:
Remove, as it's default in previous rule